src/qt/treectrl.cpp

```diff
@@ -374,7 +375,7 @@ class wxQTreeWidget : public wxQtEventSignalHandler<QTreeWidget, wxTreeCtrl>
             wxQtConvertTreeItem(current)
         );
         changedEvent.SetOldItem(wxQtConvertTreeItem(previous));
-        wxPostEvent(treeCtrl, changedEvent);
+        EmitEvent(changedEvent);
     }
```

Comment:
According to the comment above these lines, using `wxPostEvent` is intentional! but if using `EmitEvent`  here doesn't break anything, then the comment should be removed.

Reply:
Updated with another method that works on my config.